libmscore/note.cpp

```diff
@@ -761,6 +761,8 @@ QString Note::tpcUserName(bool explicitAccidental) const
       {
       QString pitchName = tpc2name(tpc(), NoteSpellingType::STANDARD, NoteCaseType::AUTO, explicitAccidental);
       QString octaveName = QString::number((epitch() / 12) - 1);
+      int alter = tpc2alterByKey(tpc(), Key::C);
+      octaveName = QString::number(((epitch() - alter) / 12) - 1);
```

Comment:
The way you have it, you are assigning a value to `octaveName` twice. My suggestion was to calculate `alter` before declaring `octaveName`, and only assigning a value to `octaveName` once.

On an unrelated note, this PR has the same problem as #5306.

Reply:
I was wondering why you redeclared octave name. I should have known the mistake was mine and not yours.